concepts/0014-didcomm-message-anatomy/README.md

```diff
@@ -0,0 +1,96 @@
+# Title (Ex. 0014: DID Comm Message Anatomy)
```

Comment:
The title here needs to match the first merge PR (0021) and remove the template text.
`# 0021: DID Comm Message Anatomy